src/main/java/rs/hybridit/model/BookCopy.java

```diff
@@ -0,0 +1,79 @@
+package rs.hybridit.model;
+
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import rs.hybridit.dto.BookCopyDto;
+
+@Entity
+@Table(catalog = "dbhybridlibrary", name = "book_copies")
+public class BookCopy {
+
+	@Id
+	@GeneratedValue
+	private Long id;
+
+	private Date rentStart;
```

Comment:
you should use java time api instead of old Date

https://www.baeldung.com/java-8-date-time-intro